tests/unit/backend/corpora/common/entities/test_dataset.py

```diff
@@ -135,3 +135,13 @@ def test__create_ids__ok(self):
 
         self.assertEqual(contributor_ids, get_ids(dataset.contributors))
         self.assertEqual(["test_contributor_id"], get_ids(dataset.contributors))
+
+    def test__list__ok(self):
+        generate = 5
+
+        for i in range(generate):
```

Comment:
```suggestion
        for _ in range(generate):
```
Do not need i here if it not used.